src/main/java/pingis/utils/JavaSyntaxChecker.java

```diff
@@ -28,4 +29,14 @@
 
     return null;
   }
+  
+  public static String[] checkTaskPairErrors(String submissionCode, String staticCode) {
+    String[] submissionSyntaxErrors = parseCode(submissionCode);
+    String[] staticSyntaxErrors = parseCode(staticCode);
+    if (submissionSyntaxErrors != null || staticSyntaxErrors != null) {
+      String[] errors = ArrayUtils.addAll(submissionSyntaxErrors, staticSyntaxErrors);
+      return errors;
+    }
+    return null;
```

Comment:
Depending on the usage it might be cleaner to just return the array and check if it's empty where used.